EIPS/eip-7800.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7800
```

Comment:
```suggestion
eip: 7801
```
Assigning next sequential EIP/ERC/RIP number.
Self assigned numbers are manually bumped by at least one to avoid perception of special treatment or number sniping.  Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
LOL